ruma-federation-api/src/serde/v1_pdu.rs

```diff
@@ -1,42 +1,48 @@
-//! A module to deserialize a RoomState struct from incorrectly specified v1
-//! send_join endpoint.
+//! A module to deserialize a membership struct from incorrectly specified v1
+//! membership endpoint.
```

Comment:
This shouldn't mention any single endpoint. Can probably just say "deserialize a struct" and "incorrectly specified endpoint" without further details.

Reply:
Perhaps something like this?

```suggestion
//! A module to deserialize an endpoint response incorrectly specified as a tuple.
```

We could use _tuple_ elsewhere for the names too.

Reply:
I don't really like 'tuple' since that does not exist in JSON. Array is ok, anyway I went with my own rephrasing and listed the endpoints. Tell me if it is acceptable.

Reply:
Seems fine. I guess there might be more endpoints like this, but the list should be useful without being exhaustive.